assets/app/view/welcome.rb

```diff
@@ -16,7 +16,8 @@ def render
 
     def render_notification
       message = <<~MESSAGE
-        <p>18 India and 18Uruguay are in alpha.</p>
+        <p>18 India and 18Uruguay are in alpha.</br>
+           18 SJ is now in production.</p>
```

Comment:
Those HTML tags are mismatched. A line break on its own should be `<br/>`, or it might be better to put these in separate paragraphs.
```suggestion
        <p>18 India and 18Uruguay are in alpha.</p>
        <p>18 SJ is now in production.</p>
```

Reply:
Made the proposed change.